src/TimeGrid.jsx

```diff
@@ -77,9 +80,24 @@ export default class TimeGrid extends Component {
       this.measureGutter()
     }
 
+    if (this._scrollRatio) {
+      const { content } = this.refs;
+      content.scrollTop = content.scrollHeight * this._scrollRatio;
+      // Only do this once
+      this._scrollRatio = null;
```

Comment:
[you're right](https://facebook.github.io/react/docs/component-specs.html#updating-componentdidupdate) does not get called for initial render. I'll add it there too.